src/com/mambu/apisdk/services/OrganizationService.java

```diff
@@ -134,6 +137,29 @@ public Currency getCurrency() throws MambuApiException {
 	}
 
 	/**
+	 * Requests a currency for a supplied currency code.
+	 * 
+	 * @param currencyCode
+	 *            the currency code for the currency to be searched in Mambu.
+	 * 
+	 * @return Currency having as currency code the currency code passed as parameter to this method call, or
+	 *         INVALID_CURRENCY_CODE error code if there is no currency having the code supplied to this method call.

```

Comment:
It's nice to have the description of the error case. Can possibly clarify that it is the exception that is thrown in the error scenario (when the error code is returned by Mambu server)